src/test/resources/test_data/overall_calculation_response/custom-examples/crs-2041-SDS40-document-example1-tranche2.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dates": {
+    "SLED": "2030-10-23",
+    "CRD": "2024-10-15",
```

Comment:
If this is a defaulting case and we are using alt-3 params i'm a bit confused as to why this isn't 2024-10-22 🤔